modules/nodes-cluster-enabling-features-console.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-enabling-features.adoc
+
+[id="nodes-cluster-enabling-features-console_{context}"]
+= Enabling feature sets using the web console
+
+You can use the {product-title} web console to enable feature sets for all the nodes in a cluster by editing the `FeatureGate` custom resource (CR).
```

Comment:
Same feedback as for the CLI module: s/enable feature sets for all the nodes/enable feature sets for all **of** the nodes/
or
/enable feature sets for all nodes/